submodules/faxbox/faxbox.js

```diff
@@ -221,6 +221,41 @@ define(function(require) {
 					} else {
 						callback(null, {});
 					}
+				},
+				phone_numbers: function(callback) {
+					if (!monster.config.whitelabel.ensure_valid_caller_id) {
+						callback(null);
+						return;
+					}
+
+					self.callApi({
+						resource: 'numbers.list',
+						data: {
+							accountId: self.accountId
+						},
```

Comment:
Since this API is paginated, this will only load the first page. of results. If a number was selected but is on the second page, it will not be select, which might be confusing for the user.

Reply:
Now this one is using `listAll` instead of `list`, is that right?